flake.nix

```diff
@@ -28,12 +28,18 @@
     flake-utils.lib.eachDefaultSystem
       (system:
         let
-          pkgs = import nixpkgs {
+
+          pkgs-unstable = import nixpkgs-unstable {
             inherit system;
           };
 
-          pkgs-unstable = import nixpkgs-unstable {
+          pkgs = import nixpkgs {
             inherit system;
+            overlays = [
+              (final: prev: {
+                cargo-udeps = pkgs-unstable.cargo-udeps;
+              })
+            ];
```

Comment:
What's the benefit of using the unstable udeps version this way instead of directly?

Reply:
Can't forget about using the right once. :D